guides/admin/oidc-adfs.md

```diff
@@ -101,7 +105,11 @@ following
 
 1. Enter a name for the claim rule.
 
-1. In the **Custom Rule** field, enter the following:
+1. In the **Custom Rule** field, enter a claim rule written in the
+   [ADFS Claim Rule Language](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/technical-reference/the-role-of-the-claim-rule-language).
+
+   The following example claim rule maps the Active Directory attributes
```

Comment:
I'm hedging a bit here because I'm no expert on these claim rules, and the Active Directory admin might have more informed opinions on what is correct for their org.